app/main.py

```diff
@@ -1,8 +1,19 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int, ) -> None:
+        self.name = name
+        self.age = age
+        self.people[name] = self
```

Comment:
Here, `self.people[name] = self` should be `Person.people[name] = self` to correctly reference the class attribute `people` instead of an instance attribute.